packages/liquid-html-parser/src/stage-2-ast.ts

```diff
@@ -754,6 +755,19 @@ export interface TextNode extends ASTNode<NodeTypes.TextNode> {
   value: string;
 }
 
+export interface LiquidDocParamNode extends ASTNode<NodeTypes.LiquidDocParamNode> {
+  name: string;
+  value: string;
+  paramDescription: LiquidDocParamDescription;
+  paramName: TextNode;
+  paramType: TextNode;
```

Comment:
Same deal, please add `/** */` to explain. Also why isn't `name` the `'param'` const?